contracts/strategies/traderJoe/strategy-joe-avax-spore.sol

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+import "../strategy-joe-farm-base.sol";
+
+contract StrategyJoeAvaxSporeLp is StrategyJoeFarmBase {
+
+    uint256 public avax_spore_poolId = 10;
+
+    address public joe_avax_spore_lp = 0x7012aE2B092F12Be1820acd5F1aed5d73e3116E6;
+    address public spore = 0x6e7f5C0b9f4432716bDd0a77a3601291b9D9e985;
+
+    constructor(
+        address _governance,
+        address _strategist,
+        address _controller,
+        address _timelock
+    )
+        public
+        StrategyJoeFarmBase(
+            avax_spore_poolId,
+            joe_avax_spore_lp,
+            _governance,
+            _strategist,
+            _controller,
+            _timelock
+        )
+    {}
+
+    // **** State Mutations ****
+
+    function harvest() public override onlyBenevolent {
```

Comment:
This makes a lot of sense given the needed changes that are occurring. This allows the farm base to remain relatively static. We gotta look at that getHarvestable functionality eventually.